tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2021 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Conftest module."""
+
+from typing import List
+
+import pytest
+
+
+def pytest_collection_modifyitems(items: List[pytest.Item]):
+    """
+    Modify items collection for pytest.
+
+    :param items: list of items
+    """
+    # Remove third party tests from integration tests
+    for item in items:
+        if "third_party" in item.nodeid:
+            if "integration" in item.keywords:
+                item.add_marker(pytest.mark.skip(reason="Skipped in integration tests"))
```

Comment:
Is this newly added function `pytest_collection_modifyitems(...)` called/used anywhere?

Reply:
Never mind, the `pytest_collection_modifyitems(...)` is predefined [pytest hook](https://docs.pytest.org/en/7.1.x/how-to/writing_hook_functions.html), so it should be picked up automatically.